generators/auth/templates/mongodb/jwt/src/components/auth/auth.controller.ts

```diff
@@ -220,6 +219,7 @@ export default class AuthController {
     const payload = {
       id: decodedUser.id,
       email: decodedUser.email,
+      role: decodedUser.role,
     };
 
     return this.authService.login(payload);
```

Comment:
why not login({id: value, email: value, role: value})? why do you need a variable for the payload?